recipes-bsp/optee-os/optee-os-iot2050_3.21.0.inc

```diff
@@ -21,8 +21,12 @@ OPTEE_PLATFORM = "k3-am65x"
 OPTEE_EXTRA_BUILDARGS = " \
     TEE_IMPL_VERSION=${PV} \
     CFG_ARM64_core=y CFG_TEE_CORE_LOG_LEVEL=2 CFG_USER_TA_TARGETS=ta_arm64 \
-    CFG_CONSOLE_UART=1 CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=1 CFG_CORE_DYN_SHM=y \
-    CFG_IN_TREE_EARLY_TAS=avb/023f8f1a-292a-432b-8fc4-de8471358067 \
+    CFG_CONSOLE_UART=1 CFG_CORE_DYN_SHM=y \
     CFG_WARN_INSECURE=n \
     CFG_REE_FS=n CFG_CORE_HEAP_SIZE=524288 \
     "
+
+OPTEE_EXTRA_BUILDARGS:append:secureboot = " \
+    CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=1 \
+    CFG_IN_TREE_EARLY_TAS=avb/023f8f1a-292a-432b-8fc4-de8471358067 \
```

Comment:
IIRC, avb is not related to secureboot, but rpmb pairing.

The rpmb pairing command 'optee_rpmb' is taking advantage of the avb persistent variable which depends on the secure storage of optee - in our case, RPMB.

Reply:
Right, we need only to permit some RPMB write from u-boot command line. I'll switch that piece to `rpmb-setup`. Or we write an EFI variable from U-Boot instead.